cypress/support/fragments/inventory/inventoryActions.js

```diff
@@ -43,20 +49,37 @@ export default {
     NewInventoryInstance.checkExpectedOCLCPresence(specialOCLCWorldCatidentifier);
   },
   verifySaveUUIDsFileName(actualName) {
-    // Check naming mask
     const expectedFileNameMask = /SearchInstanceUUIDs\d{4}-\d{2}-\d{2}T\d{2}_\d{2}_\d{2}\+\d{2}_\d{2}\.csv/gm;
     expect(actualName).to.match(expectedFileNameMask);
 
-    // Check date value
     const dateString = actualName.split('/');
     const actualDate = Date.parse(dateString[dateString.length - 1].slice(19, 38).replaceAll('_', ':'));
-    expect(actualDate).to.be.greaterThan(Date.now() - 100000);
-    expect(actualDate).to.be.lessThan(Date.now() + 100000);
+    verifyFileNameDate(actualDate);
   },
+
   verifySavedUUIDs(actualUUIDs, expectedUUIDs) {
     const formattedActualUUIDs = actualUUIDs.replaceAll('"', '').split('\n');
```

Comment:
you can rewrite this to chai strict assertion of 2 arrays in one line(may be directly in the test)